tensorflow/python/data/kernel_tests/options_test.py

```diff
@@ -201,6 +201,14 @@ def testProtoOptionsDefaultValuesRoundTrip(self):
         dataset_options_pb2.ThreadingOptions())
     self.assertProtoEquals(expected_pb, result)
 
+  @combinations.generate(test_base.default_test_combinations())
+  def testThreadingOptionsBackwardCompatibility(self):
+    options = dataset_ops.Options()
+    options.threading.max_intra_op_parallelism = 20
+    self.assertEqual(options.threading, options.experimental_threading)
```

Comment:
instead, check that both `options.threading.max_intra_op_parallelism` and `options.experimental_threading.max_intra_op_parallelism` is 20

Reply:
modified the test cases a bit.

Reply:
```
instead, check that both options.threading.max_intra_op_parallelism and options.experimental_threading.max_intra_op_parallelism is 20
```
This is valid only after ser-des. Adding a new test for that.